team.html

```diff
@@ -45,6 +45,34 @@ <h2>Past Members</h2>
     <span class="gray">(formerly a {{ person.role }})</span>
   </li>
   {% endif %} {% endfor %}
-</ul>
+</ul> -->
+
+<div class="flex flex-wrap">
```

Comment:
maybe add 'mb4' class to push down the 'editing list' section

Reply:
not sure i follow.  where do you want me to add mb-4?

Reply:
ah nevermind, i finally parsed what you meant by 'editing list' 🥇 .  see new commit below.

Reply:
looks good, could also just add it to this div